packages/server/src/middlewares/authenticate.ts

```diff
@@ -2,6 +2,16 @@ import { verifySession, verifyPassword } from "../utils/auth";
 import { Context } from "koa";
 import { getAccountByEmail } from "../db/accounts";
 
+export const insertSessionId = (ctx: Context): string | null => {
```

Comment:
This insert looks a lot like an extract.. I don’t understand

Reply:
Where it is used by the way?

Reply:
Good catch, old attempt not used anywhere :))